### PR TITLE
Fix #3582: Merge topic lesson title

### LINKED ISSUE #3582: Merge topic_lessons_title.xml into single xml file
Currently there are 4 versions of `topic_lessons_title.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is no difference between before and after UI.

### PR DESCRIPTION
**Pull request by ayaazansari**
Fixed #3582 
Merged the 4 topic_lessons_title.xml (landscape/portrait of phone and tablet )into single xml file .

### CODE REVIEW

**Comment by ayaazansari** [comment]
@rt4914 I'm getting this error when I try to open the fragment in the tablet portrait mode ,please help!
error: java.lang.IllegalArgumentException: The tag for topic_lessons_title is invalid. Received: layout-sw600dp-port/topic_lessons_title_0

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/build/classes" />
   </component>
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```
Comment (rt4914) [inline .idea/misc.xml]: We don't need any change to `.idea` file, this must have been done automatically from android studio please do not add this file to your commit. 
Note: Don't delete this file, we need this file we just don't need the changes this file contains.

File: app/src/main/res/values-land/dimens.xml [hunk 2]
```diff
@@ -307,4 +307,9 @@
   <dimen name="promoted_story_card_layout_margin_start">8dp</dimen>
   <dimen name="promoted_story_card_text_size">14sp</dimen>
   <dimen name="promoted_story_card_margin_top">8dp</dimen>
+
+  <!-- Topic Lessons Title -->
+  <dimen name="topic_lessons_title_marginStart">100dp</dimen>
```
Comment (rt4914) [inline app/src/main/res/values-land/dimens.xml]: Rename this and all other dimens name to `topic_lessons_title_margin_start `.
Basically all names should contain only small-characters separated by `_`.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Thanks.

Please add before and after screenshot for this UI.

Also, once you remove tablet related files, make sire you install the app from tablet device, rebuild the app and then install the app again on tablet and it should work correctly.

**Comment by ayaazansari** [comment]
@rt4914 after those changes I'm getting the wrong layout , the "Stories you can play" title comes to the starting without any margin..
Screenshot_1633183770

File: .idea/misc.xml (hunk 1 above)
Comment (rt4914) [reply .idea/misc.xml]: @ayaazansari PTAL

File: app/src/main/res/values-land/dimens.xml (hunk 2 above)
Comment (rt4914) [reply app/src/main/res/values-land/dimens.xml]: @ayaazansari PTAL

**Review by rt4914 — CHANGES_REQUESTED** [review]
You are getting the wrong output because the TextView `topic_play_text_view ` width is `wrap_content` in some layouts and `528dp` fixed in some layouts.
You might beed to use `ConstraintLayout` to handle is correctly.

**Comment by rt4914** [comment]
@ayaazansari PTAL

**Comment by ayaazansari** [comment]
@rt4914 what layout width should I use for port,land of phone and port,land of tablet ?

**Comment by rt4914** [comment]
> @rt4914 what layout width should I use for port,land of phone and port,land of tablet ?

[Mobile Portrait](https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/4705f47b-9ada-4159-8b81-8db32cd3737d/)
- Width = Variable
- Margins = Fixed (as mentioned in mocks)

[Mobile Landscape](https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/f0c4284e-f2a6-4052-98e2-778932290c08/specs/)
- Width = Variable
- Margins = Fixed (as mentioned in mocks)

[Tablet Portrait](https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/a98b7a3c-46c4-49c6-af77-14a1ff5b7b03/)
- Width = 528dp (as mentioned in mocks)
- Margins = Variable

[Tablet Landscape](https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/28b4ea3f-6436-4e8f-a737-28373f56525b/)
- Width = 528dp (as mentioned in mocks)
- Margins = Variable


You might need to implement `ConstraintLayout` inside `ConstraintLayout` to achieve this.

**Pull request closed without merging**